monitoring/README.md

```diff
@@ -3,41 +3,41 @@
 ## Requirements
 
 1. `forest` node running locally
-2. `docker` & `docker compose` must be available in the `$PATH` (alternatively
+2. `docker` & `docker-compose` must be available in the `$PATH` (alternatively
```

Comment:
`docker compose` is a valid subcommand for the new compose 2.x. `docker-compose` is 1.x.